src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,46 @@
+package calculator
+
+enum class Operator : MathElement {
+    Addition {
+        override val symbol = "+"
+        override fun calculate(operand1: Operand, operand2: Operand) =
+            Operand(operand1.value + operand2.value)
+    },
+    Subtraction {
+        override val symbol = "-"
+        override fun calculate(operand1: Operand, operand2: Operand) =
+            Operand(operand1.value - operand2.value)
+    },
+    Multiplication {
+        override val symbol = "*"
+        override fun calculate(operand1: Operand, operand2: Operand) =
+            Operand(operand1.value * operand2.value)
+    },
+    Division {
+        override val symbol = "/"
+        override fun calculate(operand1: Operand, operand2: Operand) =
+            Operand(operand1.value / operand2.value)
+    };
+
+    abstract val symbol: String
+
+    abstract fun calculate(operand1: Operand, operand2: Operand): Operand
+
+    companion object {
+        fun isOperatorSymbol(operatorString: String): Boolean {
+            return operatorString in symbols()
+        }
+
+        fun from(operatorString: String) = when (operatorString) {
```

Comment:
```suggestion
        fun from(operatorString: String) = values().find { it.symbol == operatorString } ?: throw ~~
```
when절을 사용하는 것도 좋지만 `find`와 엘비스 연산자를 활용해서 나타낼 수도 있겠네요.